File: pgm/ABC_Java/ABC_411_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String P = reader.readLine();
        final int L = Integer.parseInt (reader.readLine());
        reader.close();
        final String Yes = "Yes";
        final String No = "No";
        System.out.println (P.length() >= L ? Yes : No);
        System.exit (0);
    }
}

File: pgm/ABC_Kotlin/ABC_411_A.kt
// import java.time.*

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"

fun main() {
    val P: String = readln()
    val L: Int = readln().toInt()
    if (P.length >= L) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}
